### PR TITLE
UI: Facet View Enhancement

### PR DESCRIPTION
**Pull request by tito12**
### Problem

Now facets are just simple printed JSON, without ability to search, navigate and click on links.
Related to task https://github.com/MarquezProject/marquez/issues/2165

### Solution

Created dynamic component basen on forked solution and extended, with ability to search through json and navigate easy, expand data and ability to click links. 

image

### Checklist

- [x] You've [signed-off](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#sign-your-work) your work
- [ ] Your changes are accompanied by tests (_if relevant_)
- [ ] Your change contains a [small diff](https://kurtisnusbaum.medium.com/stacked-diffs-keeping-phabricator-diffs-small-d9964f4dcfa6) and is self-contained
- [ ] You've updated any relevant documentation (_if relevant_)
- [ ] You've updated the [`CHANGELOG.md`](https://github.com/MarquezProject/marquez/blob/main/CHANGELOG.md#unreleased) with details about your change under the "Unreleased" section (_if relevant, depending on the change, this may not be necessary_)
- [ ] You've versioned your `.sql` database schema migration according to [Flyway's naming convention](https://flywaydb.org/documentation/concepts/migrations#naming) (_if relevant_)
- [ ] You've included a [header](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#copyright--license) in any source code files (_if relevant_)

### CODE REVIEW

**Comment by wslulciuc** [comment]
> Created dynamic component basen on forked solution and extended

@tito12, can you provide some background as to why we are forking `react-json-view`?

**Comment by phixMe** [comment]
I'd really prefer a solution here that doesn't fork a repo. What does this look like if we just use the library out of the box entirely?

**Comment by tito12** [comment]
> I'd really prefer a solution here that doesn't fork a repo. What does this look like if we just use the library out of the box entirely?

This package not contain search functionality and clickable links, which is important due task requirments. 
One if idea is to use `patch-package`, maybe it could be more clean from code side, what do you think?

**Comment by tito12** [comment]
@phixMe please take a look, I linked github repo with fork as we discussed. Thanks!

**Comment by tito12** [comment]
@phixMe please review :)

File: web/.eslintignore [hunk 1]
```diff
@@ -1 +1 @@
-/*.*
```
Comment (phixMe) [inline web/.eslintignore:1]: What's this?

File: web/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,8 @@
 FROM node:12-alpine
 WORKDIR /usr/src/app
 RUN apk update && apk add --virtual bash coreutils
+RUN apk add --no-cache git
+RUN git --version
```
Comment (phixMe) [inline web/Dockerfile:5]: Do we need to do this? Or is this for debugging.

File: web/src/components/jobs/RunInfo.tsx [hunk 3]
```diff
@@ -30,7 +30,7 @@ const RunInfo: FunctionComponent<RunInfoProps> = props => {
           <Box mb={1}>
             <MqText subheading>{i18next.t('jobs.runinfo_subhead')}</MqText>
           </Box>
-          <MqJson code={run.facets} />
+          <MqJsonView data={run.facets} placeholder='Search' />
         </Box>
       )}
     </Box>
```
Comment (phixMe) [inline web/src/components/jobs/RunInfo.tsx:36]: Can we remove MqJson as part of this as we're moving forward with this JSON viewer implementation.

**Review by phixMe — COMMENTED** [review]
A few comments here.

File: web/Dockerfile (hunk 2 above)
Comment (tito12) [reply web/Dockerfile]: The second line was for debugging, removed it

File: web/src/components/jobs/RunInfo.tsx (hunk 3 above)
Comment (tito12) [reply web/src/components/jobs/RunInfo.tsx:36]: Sure, make sense, it's done

**Review by phixMe — APPROVED** [review]
Thanks, this looks amazing.

**Pull request merged**